### ISSUE TITLE
File preview not showing in chat

### ISSUE THREAD

**Issue by THDMoritzEnderle** [nextcloud/spreed#5411]
### Steps to reproduce
Send any non-picture files in any chat ( for example pdf )

### Expected behaviour
There should at least be the file name visible and the ability to download the file

### Actual behaviour
The chat message is blank for both, no preview, no name, no possibility to download

When uploading a file (file already selected):
![image](https://user-images.githubusercontent.com/73176174/112346073-e72f1300-8cc5-11eb-9502-e96f03fc6c2b.png)

in chat:
![image](https://user-images.githubusercontent.com/73176174/112346251-0fb70d00-8cc6-11eb-92c1-dd2acfbc985d.png)

## Server configuration detail

**Operating system:** Linux 5.8.0-48-generic #54~20.04.1-Ubuntu SMP Sat Mar 20 13:40:25 UTC 2021 x86_64

**Webserver:** Apache/2.4.41 (Ubuntu) (apache2handler)

**Database:** mysql 10.3.25

**PHP version:** 

7.4.15
Modules loaded: Core, date, libxml, openssl, pcre, zlib, filter, hash, Reflection, SPL, session, standard, sodium, apache2handler, mysqlnd, PDO, xml, apcu, bcmath, bz2, calendar, ctype, curl, dom, mbstring, FFI, fileinfo, ftp, gd, gettext, gmp, iconv, igbinary, intl, json, exif, mysqli, pdo_mysql, apc, posix, readline, shmop, SimpleXML, sockets, sysvmsg, sysvsem, sysvshm, tokenizer, xmlreader, xmlwriter, xsl, zip, Phar, Zend OPcache

**Nextcloud version:** 21.0.0 - 21.0.0.18

**Updated from an older Nextcloud/ownCloud or fresh install:** 

**Where did you install Nextcloud from:** unknown

<details><summary>Signing status</summary>

Array
(
)

</details>

<details><summary>List of activated apps</summary>

```
Enabled:
 - accessibility: 1.7.0
 - activity: 2.14.3
 - audioplayer: 3.1.0
 - breezedark: 21.0.5
 - calendar: 2.2.0
 - camerarawpreviews: 0.7.10
 - cloud_federation_api: 1.4.0
 - comments: 1.11.0
 - contacts: 3.5.1
 - contactsinteraction: 1.2.0
 - dashboard: 7.1.0
 - dav: 1.17.1
 - drawio: 1.0.0
 - federatedfilesharing: 1.11.0
 - federation: 1.11.0
 - files: 1.16.0
 - files_mindmap: 0.0.24
 - files_pdfviewer: 2.1.0
 - files_rightclick: 1.0.0
 - files_sharing: 1.13.1
 - files_trashbin: 1.11.0
 - files_versions: 1.14.0
 - files_videoplayer: 1.10.0
 - firstrunwizard: 2.10.0
 - forms: 2.2.2
 - integration_reddit: 0.0.11
 - issuetemplate: 0.7.0
 - logreader: 2.6.0
 - lookup_server_connector: 1.9.0
 - mail: 1.9.3
 - nextcloud_announcements: 1.10.0
 - notifications: 2.9.0
 - oauth2: 1.9.0
 - password_policy: 1.11.0
 - passwords: 2021.3.0
 - photos: 1.3.0
 - privacy: 1.5.0
 - provisioning_api: 1.11.0
 - recommendations: 1.0.0
 - serverinfo: 1.11.0
 - settings: 1.3.0
 - sharebymail: 1.11.0
 - spreed: 11.1.1
 - support: 1.4.0
 - survey_client: 1.9.0
 - systemtags: 1.11.0
 - tasks: 0.13.6
 - text: 3.2.0
 - theming: 1.12.0
 - twofactor_backupcodes: 1.10.0
 - updatenotification: 1.11.0
 - user_status: 1.1.1
 - viewer: 1.5.0
 - weather_status: 1.1.0
 - workflowengine: 2.3.0
Disabled:
 - admin_audit
 - afterlogic
 - encryption
 - files_external
 - richdocuments
 - richdocumentscode
 - user_ldap

```
</details>

<details><summary>Configuration (config/config.php)</summary>

```
{
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "127.0.0.1:8079",
        "192.168.1.*",
        "192.168.178.*"
    ],
    "trusted_proxies": "***REMOVED SENSITIVE VALUE***",
    "overwritehost": "cloud.enderle-solutions.de",
    "overwriteprotocol": "https",
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "dbtype": "mysql",
    "version": "21.0.0.18",
    "overwrite.cli.url": "http:\/\/localhost:8079",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbport": "",
    "dbtableprefix": "oc_",
    "mysql.utf8mb4": true,
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "installed": true,
    "memcache.local": "\\OC\\Memcache\\APCu",
    "filelocking.enabled": false,
    "log_type": "file",
    "logfile": "\/var\/log\/nextcloud.log",
    "logfilemode": 416,
    "loglevel": "0",
    "logdateformat": "F d, Y H:i:s",
    "maintenance": false,
    "updater.release.channel": "beta",
    "theme": "",
    "app_install_overwrite": [
        "richdocumentscode",
        "richdocumentscode_arm64",
        "drawio",
        "flowupload",
        "files_frommail",
        "issuetemplate"
    ]
}
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** 

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

## Client configuration

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.182 YaBrowser/21.2.4.165 Yowser/2.5 Safari/537.36

**Operating system:** 

## Logs

<details><summary>Web server error log</summary>

```
Insert your web server log here 
```
</details>

<details><summary>Nextcloud log</summary>

```
Insert your Nextcloud log here
```
</details>

<details><summary>Browser log</summary>

Insert your browser log here, this could for example include:

	a) The javascript console log
	b) The network log
	c) ...
</details>

**Comment by nickvergessen** [comment]
Mostlikely the app "breezedark" is causing this.

**Comment by THDMoritzEnderle** [comment]
Completly removed breezedark, issue is still present in dark mode and normal light mode

**Comment by nickvergessen** [comment]
Is there an error in your browser console (F12) or in the network tab of the browser console

**Comment by THDMoritzEnderle** [comment]
![image](https://user-images.githubusercontent.com/73176174/112351768-75a59380-8cca-11eb-8353-eb6b106df434.png)

In network tab:
img/notifications-new.svg -> status canceled

**Comment by nickvergessen** [comment]
Ah, its the broken camerarawpreviews version:
https://github.com/nextcloud/spreed/issues/5195#issuecomment-785771092

**Comment by THDMoritzEnderle** [comment]
fixed it, thanks :D

**Comment by Byter3** [comment]
hi there!
Same ossue here!
NC 20.0.9
This will happen, if I try to send anything but images or videos, it happens even with PDF.

**Comment by nickvergessen** [comment]
As per above disable the breezedark app.
This is not a bug in Talk

**Comment by Byter3** [comment]
Sorrz maybe I wasn't clear enugh. I never installed that breezedark app. I'm only using the default dark theme, but the bug occur even if I use it in default theme mode.

**Comment by nickvergessen** [comment]
Sorry I meant `camerarawpreviews`

**Comment by Byter3** [comment]
Yes, disabling the camera raw preview app solved my problem thank you!

**Comment by mstyp** [comment]
> Sorry I meant `camerarawpreviews`

I do not have this app or the breezedark app installed. Same issue, although in the browser console is different then THDs above. `TypeError: n.includes is not a function` instead of cannot read properties. 

I tried disabling a whole bunch of apps, and that fixed the issue. But then I went through and enabled all the apps I considered "essential", and the issue came back. Have not spent time to go through each one one by one to see which is causing the break, will update this comment / add a new one if I figure it out.
